lib/internal/Magento/Framework/Model/EntitySnapshot.php

```diff
@@ -65,7 +66,7 @@ public function registerSnapshot($entityType, $entity)
      * @param object $entity
      * @return bool
      */
-    public function isModified($entityType, $entity)
+    public function isModified($entityType, $entity): bool
```

Comment:
Unfortunately, we are not allowed to change public method signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)